gamest_plugins/process_identifier/module.py

```
@@ -141,22 +154,32 @@ def candidates(self):
+        survivors = {(p.pid, p.info['create_time']): p for p in self._iter_survivors()}
+
+        self._checked_procs &= survivors.keys()
+
+        candidates = []
+        for key, p in survivors.items():
+            if key in self._checked_procs:
+                continue
+            try:
+                with p.oneshot():
+                    candidates.append((p, p.exe(), p.cmdline()))
+            except Exception:
```

Pylint (4.0.5)

[warning] Catching too general exception Exception

(W0718)